ipatests/test_integration/test_backup_and_restore.py

```diff
@@ -526,19 +531,31 @@ def test_userroot_ldif_files_ownership(self):
         self.master.run_command(['ipa-restore', backup_path],
                                 stdin_text=dirman_password + '\nyes')
 
+        result = self.master.run_command(['umask'])
+        assert '0022' in result.stdout_text
+
         # check if files have proper owner and group.
         dashed_domain = self.master.domain.realm.replace(".", '-')
         arg = ['stat',
-               '-c', '%U%G',
+               '-c', '%U:%G',
                '/var/lib/dirsrv/slapd-' + dashed_domain + '/ldif']
         cmd = self.master.run_command(arg)
-        assert 'dirsrvdirsrv' in cmd.stdout_text
+        assert 'dirsrv:dirsrv' in cmd.stdout_text
+
+        # also check of access rights are set to 644
+        arg = ['stat',
+               '-c', '%U:%G:%a',
+               ('/var/lib/dirsrv/slapd-{}/ldif/'
```

Comment:
nit: paths.VAR_LIB_SLAPD_INSTANCE_DIR_TEMPLATE is a built-in instead of hardcoding the path name to the instance in /var.

Similarly this shouldn't hardcode the dirsrv login name, use DS_USER.

I suppose it is questionable that these files remain after a restore at all.